Assignment2/jmdu99-JoseManuelDiaz.ttl

```diff
@@ -0,0 +1,10 @@
+@base <http://example.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <http://example.org/components/> .
+:Class01  	:includes :Sensor029;		
+                :includes :Computer101.
+:Sensor029  	:hasMeasurement :Measurement8401.
+:Computer101	:hasOwner :User10A.
+:Measurement8401 :hasTemperature "29" ^^xsd:integer;
```

Comment:
Just take into account that I am not sure whether every single parser will parse a string where there is a blank space before the datatype declaration. Anyway, accepted

Reply:
Okey, i'll bear that in mind. Thank you.